pcsx2/Gif.cpp

```diff
@@ -85,9 +192,33 @@ __fi void gifInterrupt()
 		return;
 	}	
 
+	if (CHECK_GIFFIFOHACK) {
+		
+		if (int amtRead = gif_fifo.read(true)) {
+
+			if (!gifUnit.Path3Masked() || gifRegs.stat.FQC < 16) {
+				GifDMAInt(amtRead * BIAS);
+				return;
+			}
+		}
+		else {
+
+			if (!gifUnit.CanDoPath3() && gifRegs.stat.FQC == 16)
+			{
+				if (gifch.qwc > 0 || gspath3done == false) {
+					if (!gifUnit.Path3Masked()) {
```

Comment:
``qwc`` is an ``u16`` , ``gifch.qwc > 0`` could just be ``gifch.qwc``. I also prefer ``!gspath3done`` over ``gspath3done == false``